src/components/TextField/__stories__/TextField.stories.mdx

```diff
@@ -84,80 +103,99 @@ There are three sizes available: Small (32px), Medium (40px) and Large (48px).
       </div>
       <div className="monday-storybook-text-field_column-wrapper">
         <TextField placeholder="With field label" title="Name" size={TextField.sizes.MEDIUM} />
-        <TextField placeholder="Success" validation={{ status: "success" }} iconName={Check} size={TextField.sizes.MEDIUM} />
-        <TextField placeholder="Error" validation={{ status: "error" }} iconName={CloseSmall} size={TextField.sizes.MEDIUM} />
+        <TextField
+          placeholder="Success"
+          validation={{ status: "success" }}
+          iconName={Check}
+          size={TextField.sizes.MEDIUM}
+        />
+        <TextField
+          placeholder="Error"
+          validation={{ status: "error" }}
+          iconName={CloseSmall}
+          size={TextField.sizes.MEDIUM}
+        />
       </div>
     </div>
   </Story>
 </Canvas>
 
```

Comment:
let's move the story with the description example to be part of the variants stories